sources/osg/FrameBufferObject.js

```diff
@@ -252,6 +252,10 @@ FrameBufferObject.prototype = MACROUTILS.objectInherit( GLObject.prototype, MACR
         if ( status !== 0x8CD5 ) {
             this._reportFrameBufferError( status );
         }
+        if ( gl.checkFramebufferStatus( gl.FRAMEBUFFER ) !== gl.FRAMEBUFFER_COMPLETE ) {
```

Comment:
Yep you are right, but I don't get why to compare against the HexValue and not to the alias, which seems more readable. I will change it.